src/pools/composableStable/composableStablePool.ts

```diff
@@ -180,19 +180,23 @@ export class ComposableStablePool extends PhantomStablePool {
         const balancesEvm = this.tokens
             .filter((t) => !isSameAddress(t.address, this.address))
             .map(({ balance, priceRate, decimals }) =>
-                parseFixed(balance, 18)
```

Comment:
This should be left at 18. It should be same as smart contracts where all values are normalized as if its balance had 18 decimals.

Reply:
This diff is actually not changing anything in the result. I simply switched `decimals` and `18` on both lines. It was more of a conceptual change, because we should parse the balance by their decimals and the priceRate by 18, instead of the opposite. 😅